.github/workflows/release.yml

```diff
@@ -101,14 +97,34 @@ jobs:
       - name: "Read changelog from draft release"
         run: gh release view ${{ env.PACKAGE_VERSION }} --json body --template '{{ .body }}' >> changelog
 
-      # TODO: Sign tag
-      - name: "Create release tag"
-        run: git tag -a -F changelog ${{ env.PACKAGE_VERSION }}
-
-      - name: "Update version information to next patch development release"
-        run: |
-          ./bin/update-release-version.php to-next-patch-dev
-          git commit -m "Back to -dev" phongo_version.h
+      # This step creates the "Package x.y.z" commit that will be the base of
+      # our tag and creates the release tag. This is run inside the container in
+      # order to create signed git artifacts
+      - name: "Create package commit and release tag"
+        uses: ./.github/actions/garasign/git-sign
+        with:
+          command: "$(pwd)/.github/workflows/commit-and-tag.sh ${{ env.PACKAGE_VERSION }} ${{ vars.GPG_KEY_ID }}"
```

Comment:
If you end up parameterizing the `TAG_MESSAGE` filename, you can explicitly pass it here. I won't advocate for making "changelog" a variable in this file, as everything is self-contained.

----

I see the changelog is generated from [gh_release_create](https://cli.github.com/manual/gh_release_create). Does that become the entire body of the release draft?

This is a departure from the current git tag message we use (e.g. "Release X.Y.Z"). Considering we'll probably end up revising the body of the release notes, I have mixed feelings about preserving unedited content in the tag. Perhaps that's not a concern if this changelog is entirely derived from existing, immutable commit messages.

Reply:
Ah, good point.

The release draft will contain the original release message as well as the full changelog generated from GitHub. The idea for the tag was to use the original title and place the full changelog in the tag body. I've now fixed this as I previously only used the changelog for the tag message.